middle_end/flambda/simplify_primitives.ml

```diff
@@ -123,6 +123,7 @@ let primitive (p : Clambda_primitives.primitive) (args, approxs)
   | Pmakearray(_, _) when is_empty approxs ->
     Prim (Pmakeblock(0, Asttypes.Immutable, Some []), [], dbg),
     A.value_block (Tag.create_exn 0) [||], C.Benefit.zero
+  (* CR mshinwell: Work out what to do here with [Pmakefloatblock] *)
```

Comment:
Do you plan something more complicated than treating it the same as Pfloatarray here?

Reply:
No, this was also a stale CR.  Flambda 1 uses the Clambda primitives in its IR, so in fact it never sees `Pmakefloatblock` here, as `Convert_primitives` has turned it into `Pmakearray Pfloatarray`.